projects/js-packages/partner-coupon/package.json

```diff
@@ -27,6 +27,7 @@
 		"@automattic/jetpack-connection": "workspace:^0.13.1-alpha",
 		"@automattic/jetpack-components": "workspace:^0.10.1-alpha",
 		"@wordpress/i18n": "4.2.4",
+		"@babel/preset-react": "7.16.0",
```

Comment:
Should we keep these in alphabetical order?

Reply:
I tried at first bu realized there's no consistency across packages. I can double check

Reply:
Fixed the dependency on this package. It was not under `devDependencies`.

> I note @babel/preset-react has a peer dependency on @babel/core that's not being fulfilled in most of these now.

Where do you see it? Anything we need to do about it?

Reply:
> Where do you see it?

Nowhere very obvious, unfortunately. The addition in the pnpm-lock.yaml at https://github.com/Automattic/jetpack/pull/22359/files#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR3090-R3102 pointed me towards looking for it.

Someday I hope to clean up all the missing peer deps and then enable the configuration to enforce them going forward, but I have a bunch of other stuff on my todo list first.

> Anything we need to do about it?

Add a dependency on `@babel/core` alongside the added dependency on `@babel/preset-react`. Same version number should be good.

Except for js-packages/storybook, since that already has a `@babel/core` dep.

Reply:
done